### PR TITLE
High school tasks break down

### PR DESCRIPTION
**Pull request by Ansper11**

### CODE REVIEW

File: High school task break down.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Milestone description: In this milestone we will primarily focus on the environment of the lesson. The deliverable in peer test 1 is a lesson prototype based on Minecraft platform.
```
Comment (Nick-McGee) [inline High school task break down.txt:1]: This is a good goal to set!

File: High school task break down.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Milestone description: In this milestone we will primarily focus on the environment of the lesson. The deliverable in peer test 1 is a lesson prototype based on Minecraft platform. 
+In this prototype, User should be able to load our lesson in Minecraft Education (Map and lesson tasks), and they should be able to communicate with Non-player characters (NPC) and get a simple task. 
+Users should be able to write and run python code in the build-in code editor. Due to the time constraint, we can only finish a part of the linear regression scenario. In this part we will teach users some basic knowledge of linear regression.
```
Comment (Nick-McGee) [inline High school task break down.txt:3]: I agree, we should only teach the basic knowledge of linear regression

**Comment by eloiseespel** [comment]
I like the idea of the breakdown very much appropriate

**Comment by eloiseespel** [comment]
Also good that you included Python in that it will likely be the heart of our coding and the student's coding

**Comment by yyht6300a** [comment]
Thanks for your work.

**Pull request merged**